doc/source/set_operations.rst

```diff
@@ -19,58 +19,99 @@ The basic idea is demonstrated by the graphic below but keep in mind that overla
 
 
 Overlay Example
------------------
+---------------
 
-First, we load some example data:
+First, we create some example data:
 
 .. ipython:: python
 
-    world = gpd.read_file(gpd.datasets.get_path('naturalearth_lowres'))
-    capitals = gpd.read_file(gpd.datasets.get_path('naturalearth_cities'))
+    from shapely.geometry import Polygon
+    polys1 = gpd.GeoSeries([Polygon([(0,0), (2,0), (2,2), (0,2)]),
+                            Polygon([(2,2), (4,2), (4,4), (2,4)])])
+    polys2 = gpd.GeoSeries([Polygon([(1,1), (3,1), (3,3), (1,3)]),
+                            Polygon([(3,3), (5,3), (5,5), (3,5)])])
 
-    # Select some columns
-    countries = world[['geometry', 'name']]
+    df1 = gpd.GeoDataFrame({'geometry': polys1, 'df1':[1,2]})
+    df2 = gpd.GeoDataFrame({'geometry': polys2, 'df2':[1,2]})
 
-    # Project to crs that uses meters as distance measure
-    countries = countries.to_crs('+init=epsg:3395')[countries.name!="Antarctica"]
-    capitals = capitals.to_crs('+init=epsg:3395')
+These two GeoDataFrames have some overlapping areas:
 
-To illustrate the ``overlay`` function, consider the following case in which one wishes to identify the "core" portion of each country -- defined as areas within 500km of a capital -- using a ``GeoDataFrame`` of countries and a ``GeoDataFrame`` of capitals.
+.. ipython:: python
+
+    ax = df1.plot(color='red');
+    @savefig overlay_example.png width=5in
+    df2.plot(ax=ax, color='green');
+
+
+We illustrate the different overlay modes with the above example. First, with
+``how='intersection'``, it returns the geometries shared by both GeoDataFrames:
+
+.. ipython:: python
+
+    res_intersection = gpd.overlay(df1, df2, how='intersection')
+    res_intersection
+
+    ax = res_intersection.plot()
+    df1.plot(ax=ax, facecolor='none');
+    @savefig overlay_example_intersection.png width=5in
+    df2.plot(ax=ax, facecolor='none');
+
+Using ``how='union'``, all geometries from combining both GeoDataFrames are
```

Comment:
Hahaha. Right. How about...

When ``how='union'``, ``overlay`` returns the set of all individual geometries from the two input GeoDataFrames. This result covers the area covered by the two input GeoDataFrames, and also preserves all unique regions defined by combined boundaries of the two GeoDataFrames.